boards/nz32-sc151/Makefile.include

```diff
@@ -0,0 +1,25 @@
+## the cpu to build for
+export CPU = stm32l1
+export CPU_MODEL = stm32l151rc
+
+# define the default port depending on the host OS
+PORT_LINUX ?= /dev/ttyUSB0
+PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.SLAB_USBtoUART*)))
+
+# set the default id
+export ID ?= 0483:df11
+
+export BINFILE = $(patsubst %.elf,%.bin,$(ELFFILE))
+
+export FLASHER = dfu-util
+export DEBUGGER = # spark core has no debugger
```

Comment:
Well as it's configured by default for using dfu-util, this variable should be kept empty (but maybe I'm wrong)